Sources/MapboxCoreNavigation/NavigationService.swift

```diff
@@ -57,18 +57,18 @@ public protocol NavigationService: CLLocationManagerDelegate, RouterDataSource,
     var eventsManager: NavigationEventsManager { get }
     
     /**
-     The route along which the user is expected to travel, plus its index in the `RouteResponse`, if applicable.
+     The route along which the user is expected to travel.
 
      If you want to update the route, use `Router.updateRoute(with:routeOptions:)` method from `router`.
      */
-    var indexedRoute: IndexedRoute { get }
-
+    var route: Route? { get }
```

Comment:
Why this because optional? Seems wrong that navigation service can operation without a route.

Reply:
That's right, it's a fallout of early implementation version